aten/src/ATen/cpu/vec256/vec256_int.h

```diff
@@ -184,6 +184,12 @@ struct Vec256<int32_t> : public Vec256i {
       std::memcpy(ptr, tmp_values, count * sizeof(int32_t));
     }
   }
+  void dump() const {
+      for (size_t i = 0; i < size(); ++i) {
+          std::cout << (int)((value_type*)&values)[i] << " ";
+      }
+      std::cout << std::endl;
+  }
```

Comment:
We should probably just go ahead and add dump() to all Vec256 variants. Can be a separate PR